source/_integrations/trend.markdown

```diff
@@ -19,6 +21,8 @@ at least two updates of the underlying sensor to establish a trend.
 Thus it can take some time to show an accurate state. It can be useful
 as part of automations, where you want to base an action on a trend.
 
+{% include integrations/config_flow.md %}
+
 ## Configuration
```

Comment:
The config flow include will have a heading called "Configuration" as well. We should make clear that this helper can be configured both via the UI as helper and manually via YAML.

Reply:
I have made some changes, what do you think?